<!--Launch sentis_m100_node to get data from Sentis ToF m100
    and send it as a point cloud topic.
    We do NOT use the sentis_tof_m100 package.
    To check IP, go http://192.168.1.1/-->
<launch>
  <node name="sentis_m100" pkg="pr2_lfd_vision" type="sentis_m100_node">
    <param name="init_fps" value="1" />
    <param name="tcp_ip" value="192.168.111.20" />
    <!--<param name="tcp_ip" value="192.168.1.147" />-->
    <param name="udp_ip" value="224.0.0.1" />
    <param name="tcp_port" value="10001" />
    <param name="udp_port" value="10002" />
  </node>
</launch>
